core/src/Cache/Items/MsalAccessTokenCacheItem.cs

```diff
@@ -45,7 +45,7 @@ internal MsalAccessTokenCacheItem()
         internal MsalAccessTokenCacheItem
             (string environment, string clientId, MsalTokenResponse response, string tenantId) :
 
-            this(environment, clientId, response.TokenType, ScopeHelper.ConvertStringToLowercaseSortedSet(response.Scope).AsSingleString(),
+            this(environment, clientId, response.TokenType, response.Scope,
```

Comment:
>response.Scope [](start = 60, length = 14)

Does this handle sorting and normalizing? I believe that is a requirement we wanted to ensure happened for the cache?

Reply:
according to schema scopes should be stored as returned from sts

Reply:
great, that was not how I understood it. Are we sure the server is sorting etc.?

---
In reply to: [227981415](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1350#discussion_r227981415) [](ancestors = 227981415)